runtime/types.go

```diff
@@ -23,3 +23,14 @@ type Version struct {
 	Spec_version      int32
 	Impl_version      int32
 }
+
+var (
+	CoreVersion                               = "Core_version"
+	CoreExecuteBlock                          = "Core_execute_block"
+	TaggedTransactionQueueValidateTransaction = "TaggedTransactionQueue_validate_transaction"
+	BabeApiConfiguration                      = "BabeApi_configuration"
+)
+
+func GetWASMConfiguration(rt *Runtime, configuration string, loc int32, data []byte) ([]byte, error) {
```

Comment:
this might be better as a receiver function 
` func (rt *Runtime) GetWASM(args...) ([]byte, error)`

also I don't think the function name is the best description of what this does, maybe just `Call` would be a better name?

Reply:
I was going to suggest this be a part of `Exec`, this seems like an unnecessary level of redirection.

Reply:
I am not sure what is the result of your discussion, what should I do ?

Reply:
we think it would be best to add it as part of the `Exec` function, does that sound reasonable?

Reply:
Right, so remove the function GetWASMConfiguration and duplicate Exec multiple times across the code and do not re-use a method with clearer definition in favor of using just the Exec ?

Reply:
no, we mean just passing the configuration string directly into Exec. since this function only calls Exec, we might as well pass `configuration` directly into `Exec`.

eg: from core/runtime.go: `ret, err := s.rt.Exec(runtime.TaggedTransactionQueueValidateTransaction, loc, e)`

Reply:
Cool, I'll change it then.

Reply:
done